src/content/docs/browser/browser-monitoring/browser-pro-features/upload-source-maps-un-minify-js-errors.mdx

```diff
@@ -17,30 +17,23 @@ import browserReleaseIds from 'images/browser_screenshot-full_release-ids.webp'
 
 [Browser monitoring](/docs/browser/new-relic-browser/getting-started/introduction-new-relic-browser) supports the uploading of source maps.
 
-Source maps support is primarily useful for "decoding" minified JavaScript. Minified JavaScript results in mostly useless error stack traces on browser's [JS errors page](/docs/browser/new-relic-browser/browser-pro-features/javascript-errors-page-examining-errors-over-time). Uploading source maps converts these errors to understandable stack traces, with useful references to code lines. This feature might also be useful for bundled or transpiled JavaScript code.
+Source maps support is primarily useful for "decoding" minified JavaScript. Minified JavaScript results in mostly useless error stack traces on browser's **Errors** page. Uploading source maps converts these errors to understandable stack traces, with useful references to code lines. This feature might also be useful for bundled or transpiled JavaScript code.
 
 Browser gives you two options for using source maps:
 
 1. Upload source maps via the browser UI
 2. Publish source maps to browser via the API
 
-This document will explain the first method: how to upload source maps via the UI, along with general [troubleshooting help](#troubleshoot). For instructions on using the API method, see [Push source maps via the API](/docs/push-source-maps-api).
+This document will explain the first method: how to upload source maps via the UI, along with general [troubleshooting help](#troubleshoot). For instructions on using the API method, see [Push source maps via the API](/docs/browser/new-relic-browser/browser-pro-features/upload-source-maps-api/).
 
 ## Import source maps via the New Relic UI [#source-map-ui]
 
-You can drag and drop, or upload, a source map file into the browser UI to associate it with a specific JavaScript file. New Relic will then convert minified stack traces into un-minified traces and source code visible on the [JS errors page](#examine-unminified).
+You can drag and drop, or upload, a source map file into the browser UI to associate it with a specific JavaScript file. New Relic will then convert minified stack traces into un-minified traces and source code visible on the Errors page.
```

Comment:
The formatting for the first part of this line is a little hard to parse. Maybe just "You can either drag and drop or upload a source map" and ditch the commas?

Also, should the Errors in Errors page be bold? I'm having a brain fog attack so I can't remember.